cmd/skycoin/skycoin.go

```diff
@@ -293,6 +297,11 @@ var devConfig = Config{
 	// Wallets
 	WalletDirectory: "",
 
+	// Timeout settings for http.Server
+	ReadTimeout:  5,
```

Comment:
should multiply by time.Second here, not later

Reply:
Ok, thanks